testfixtures/tests/test_tempdir.py

```diff
@@ -110,3 +110,15 @@ def test_method(d):
             test_method()
 
             self.assertFalse(m.called)
+
+    @tempdir()
+    def test_path_end_seperator(self, d):    
+        d.write('source/foo/bar.txt', '')
+
+        d.compare(path='source/', expected=['foo/', 'foo/bar.txt']) 
+
+    @tempdir()
+    def test_path_not_end_seperator(self, d):    
+        d.write('source/foo/bar.txt', '')
+
+        d.compare(path='source', expected=['foo/', 'foo/bar.txt']) 
```

Comment:
Are you referring to `test_path_end_seperator` or both test? 
Would you prefer I merge the test into an existing test function?